src/components/Dropdown/Dropdown.stories.mdx

```diff
@@ -0,0 +1,80 @@
+import { Meta, ColorPalette, ColorItem, ArgsTable, Story, Canvas } from '@storybook/addon-docs';
+import Dropdown from '.';
+
+<Meta
+    title="Components/Dropdown"
+    component={Dropdown}
+    argTypes={{ onClick: { action: 'clicked' } }}
+/>
+
+export const StorybookDropdown = (args) => <Dropdown {...args} />;
+
+# Dropdown
+
+Drowdown allows to show interactive content.
```

Comment:
typo, Drowdown -> Dropdown

Reply:
Hello. Thank you very much for the review. I fixed that typo.